entity-framework/core/miscellaneous/testing/in-memory.md

```diff
@@ -1,64 +1,23 @@
 ---
 title: Testing with InMemory - EF Core
-author: rowanmiller
+author: ajcvickers
+description: Using the EF in-memory database to test an EF Core application
 ms.date: 10/27/2016
-ms.assetid: 0d0590f1-1ea3-4d5c-8f44-db17395cd3f3
 uid: core/miscellaneous/testing/in-memory
 ---
 
-# Testing with InMemory
+# Using the EF in-memory database to test an EF Core application
 
-The InMemory provider is useful when you want to test components using something that approximates connecting to the real database, without the overhead of actual database operations.
+> [!WARNING]
+> The in-memory database often behaves differently than common relational databases.
```

Comment:
Consider calling it always InMemory to differentiate from SQLite in-memory

Reply:
Changed to consistently use, "EF in-memory database".